packages/button/src/styledComponents/Container.js

```diff
@@ -1,16 +1,27 @@
 // @flow
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 const Container = styled.div`
   position: relative;
-  display: ${props => (props.fluid ? "block" : "inline-block")};
```

Comment:
is the default display value (inline-block) set somewhere else?

Reply:
don't think so... is it necessary anywhere?